target/linux/ipq40xx/image/generic.mk

```diff
@@ -682,7 +682,7 @@ define Device/linksys_ea6350v3
 	IMAGE_SIZE := 35840k
 	UBINIZE_OPTS := -E 5
 	IMAGES += factory.bin
-	IMAGE/factory.bin := append-kernel | append-uImage-fakehdr filesystem | pad-to $$$$(KERNEL_SIZE) | append-ubi | linksys-image type=EA6350v3
+	IMAGE/factory.bin := append-kernel | append-uImage-fakehdr filesystem | pad-to $$$$(KERNEL_SIZE) | append-ubi | check-size | linksys-image type=EA6350v3
```

Comment:
It doesn't make sense to check the size of the ubi image as ubifs needs 20 reserved PEBs to handle the NAND bad block.

Reply:
Could you elaborate? How is the bad-block handling related to checking whether a user is building a kernel+filesystem image too large for their device? How then would such a mistake get caught or play out in practice?

Reply:
Because 20 blocks are reserved by UBI for bad block handling by default so you have 20 blocks less of free space on every UBI volume.

Reply:
Will comment in the main thread since I'm about to push some large changes, but I've lately dropped this "check-size".